spec/models/category_spec.rb

```diff
@@ -4,4 +4,66 @@
   it "can have many videos" do
     should have_many(:videos)
   end
+
+  describe "#recent_videos" do
+    let!(:category_1) { Category.create(name:"category_1") }
+    let!(:category_2) { Category.create(name:"category_2") }
+    let!(:category_3) { Category.create(name:"category_3") }
+    let!(:video_1) { Video.create(title:"1_The Godfather",
```

Comment:
You are passing a block to `let` so in Ruby multiple lines block is `do` / `end`.